### PR TITLE
Added content about when kubectl command hit from the terminal.

### PR DESCRIPTION
**Pull request by punni1403**

### CODE REVIEW

**Comment by sinni19** [comment]
Looks like content copied from https://github.com/jamiehannaford/what-happens-when-k8s. Anyways good to add in repo.

File: README.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ learn concepts around the multiple technology which crisp explaination
 - [Understanding node state management in Kubernetes.](kubernetes/understanding_node_state_management_in_kubernetes.md)
 - [What happens when we delete a pod in Kubernetes cluster.](kubernetes/What_Happens_When_Deleting_a_Pod.md)
 - [Kubernetes Scheduling - breakdown of internal mechanism.](kubernetes/Kubernetes_scheduling_breakdown_of_internals.md)
+- [What happens when k8s client hit the kubectl apply command.](kubernetes/what_happens_when_k8s_fire_kubectl_command.md)
```
Comment (sinni19) [inline README.md:10]: display language is not correct. And please add ref image for workflow.

**Review by sinni19 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (punni1403) [reply README.md:10]: changed

**Review by sinni19 — APPROVED** [review]
Looks good

**Pull request merged**